service/dap/server_test.go

```diff
@@ -2279,7 +2250,7 @@ func TestSetBreakpoint(t *testing.T) {
 					client.ExpectContinueResponse(t)
 					client.ExpectStoppedEvent(t)
 					checkStop(t, client, 1, "main.loop", 8)
-					client.VariablesRequest(1001) // Locals
+					client.VariablesRequest(localsScope) // Locals
```

Comment:
I think we don't need comments like  '// Locals' any more

Reply:
done.